contexts/utopia-natcert/v1.json

```
@@ -0,0 +1,9 @@
+{
+  "@context": {
+    "@protected": true,
+    "confidenceMethod": "https://examples.vcplayground.org/contexts/utopia-natcert/vocab#confidenceMethod",
```

Align with VC 2 spec.  Not sure if this should be in another file similar to `renderMethod`.
```suggestion
    "confidenceMethod": {
      "@id": "https://www.w3.org/2018/credentials#confidenceMethod",
      "@type": "@id"
    },
```

---

Fixed in c426eb7

---

The `confidenceMethod` URL remains an example one in the above commit.

---

Finally fixed in [9a3a7df](https://github.com/credential-handler/vc-examples/pull/76/commits/9a3a7dfe722761c6969db3039ec690e5ec8266b9)